src/watcher.rs

```diff
@@ -15,6 +15,9 @@ async fn async_watch(
     after_build: Option<String>,
 ) -> notify::Result<()> {
     loop {
+        // We want to sleep for a little while so the CPU can schedule other work. That way we end
+        // up not burning CPU cycles.
+        Delay::new(Duration::from_millis(50)).await;
```

Comment:
If we think `50ms` is too much we could lower it down to `10ms` or something like that. With `10ms` CPU stays at ~0.5%.